docs/Sway and wlroots support.md

```diff
@@ -29,6 +42,13 @@ To ensure that Flameshot is correctly positioned on multiple outputs (monitors)
 for_window [app_id="flameshot"] border pixel 0, floating enable, fullscreen disable, move absolute position 0 0
 ```
 
+and add the following on your River config:
+
+```
+riverctl float-filter-add "flameshot"
```

Comment:
This rule is not support since River 0.3.x. You should use `riverctl rule-add -app-id "flameshot"` float instead

Reply:
Okay, I'll change this

Reply:
Changed :+1: